setup.py

```diff
@@ -27,35 +27,47 @@
 import os
 import sys
 
-from setuptools import setup
+from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
-requirements = [
-    "Flask>=0.10.1",
-    "Flask-Babel>=0.9",
-    "Flask-Breadcrumbs>=0.2",
-    "Flask-Cache>=0.12",
-    "Flask-Registry>=0.2",
-    'jsonpointer>=1.9',
-    'jsonschema>=2.5.0',
-    'six>=1.7.2',
-    'speaklater>=1.3',
-    'invenio-base>=0.2.1',
-]
-
-test_requirements = [
-    'coverage>=4.0.0',
-    'pytest-cov>=2.1.0',
+tests_require = [
+    'check-manifest>=0.25',
+    'coverage>=4.0',
+    'isort>=4.2.2',
+    'pep257>=0.7.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
+    'mock>=1.3.0',
 ]
 
+extras_require = {
+    'docs': [
+        "Sphinx>=1.3",
+    ],
+    'tests': tests_require,
+}
 
-class PyTest(TestCommand):
+extras_require['all'] = []
+for reqs in extras_require.values():
+    extras_require['all'].extend(reqs)
+
+setup_requires = [
+    'Babel>=1.3',
```

Comment:
Is Babel+Flask-BabelEx required? if not, then I think we can just remove them from the dependencies.